includes/Core/Authentication/Authentication.php

```diff
@@ -1253,7 +1254,7 @@ private function require_user_input() {
 
 		// Refresh user input settings from the proxy.
 		// This will ensure the user input state is updated as well.
-		$this->user_input_settings->set_settings( null );
+		$this->user_input->set_answers( null );
```

Comment:
I know we have a separate issue to tackle this, but I was wondering, does this wipe out the settings answers in the database on every refresh at the moment? If so, may be worth having a note about it somewhere.

Reply:
It doesn't wipe out on every refresh, but it does get wiped out on the `googlesitekit_authorize_user` hook, i.e. once a user signs into Site Kit. It should be removed as a part of #5900. I have added a note for it in the QA Brief and it the PR description.